srcpkgs/kicad/template

```diff
@@ -1,26 +1,26 @@
 # Template file for 'kicad'
 pkgname=kicad
-version=5.1.11
-revision=4
+version=6.0.1
+revision=1
 build_style=cmake
 build_helper=cmake-wxWidgets-gtk3
-configure_args="-DKICAD_BUILD_VERSION=${version} -DKICAD_SCRIPTING=ON
- -DKICAD_SCRIPTING_MODULES=ON -DKICAD_SCRIPTING_WXPYTHON=ON
- -DKICAD_SCRIPTING_ACTION_MENU=ON  -DBUILD_GITHUB_PLUGIN=ON -DKICAD_USE_OCE=OFF
+configure_args="-DKICAD_SCRIPTING=ON -DKICAD_SCRIPTING_MODULES=ON
+ -DKICAD_SCRIPTING_WXPYTHON=ON -DKICAD_SCRIPTING_ACTION_MENU=ON
  -DKICAD_SCRIPTING_PYTHON3=ON -DKICAD_SCRIPTING_WXPYTHON_PHOENIX=ON
```

Comment:
Kicad always requires python 3 now and phoenix shouldn't have to be specified. I think these options should be removed.